book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(date, self, person)
+    @rentals << rental
+    person.rentals << rental unless person.rentals.include?(rental)
```

Comment:
[OPTIONAL]

- This is not required since we have already assigned the person to the rental class.